src/tools/rbd/action/Group.cc

```diff
@@ -830,12 +819,15 @@ int execute_group_snap_list(const po::variables_map &vm,
       std::ostringstream oss;
       oss << type_string;
       if (get_mirror_res == 0) {
+        if(mirror_snap.complete) {
+          copy_status = "copied";
+        }
         oss << " (" << mirror_snap_state << " "
                     << "peer_uuids:[" << mirror_snap.mirror_peer_uuids << "]";
         if (mirror_snap.state == RBD_SNAP_MIRROR_STATE_NON_PRIMARY ||
             mirror_snap.state == RBD_SNAP_MIRROR_STATE_NON_PRIMARY_DEMOTED) {
           oss << " " << mirror_snap.primary_mirror_uuid << ":"
-              << mirror_snap.primary_snap_id;
+              << mirror_snap.primary_snap_id << ": " << copy_status;
```

Comment:
There isn't a colon between the primary snap ID and the copied status in `rbd snap ls --all` output:
```
          if (mirror_snap.state == RBD_SNAP_MIRROR_STATE_NON_PRIMARY ||
              mirror_snap.state == RBD_SNAP_MIRROR_STATE_NON_PRIMARY_DEMOTED) {
            oss << " " << mirror_snap.primary_mirror_uuid << ":"
                << mirror_snap.primary_snap_id << " ";
            if (!mirror_snap.complete) {
```
```suggestion
              << mirror_snap.primary_snap_id << " " << copy_status;
```